MultiplayerMod/Multiplayer/Objects/MultiplayerObjects.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System.Collections.Generic;
 using MultiplayerMod.Core.Dependency;
+using MultiplayerMod.Core.Logging;
 using UnityEngine;
 
 namespace MultiplayerMod.Multiplayer.Objects;
 
 public class MultiplayerObjects {
 
+    private static readonly Core.Logging.Logger log = LoggerFactory.GetLogger(typeof(MultiplayerObjects));
```

Comment:
With non-static you can use just `GetLogger<T>`

Reply:
Done